api/internal/urls.py

```diff
@@ -59,7 +56,6 @@
     path("user", CurrentUserView.as_view(), name="current-user"),
     path("slack/", include("api.internal.slack.urls")),
     path("charts/", include("api.internal.chart.urls")),
-    path("", include(plans_router.urls)),
```

Comment:
Can you confirm this is not in the public api?

Reply:
It is not